support.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Required tools
+DOCKER="${DOCKER:-docker}"
+NSENTER="${NSENTER:-nsenter}"
+BRIDGE="${BRIDGE:-bridge}"
+BRCTL="${BRCTL:-brctl}"
+IPTABLES="${IPTABLES:-iptables}"
+
+NSDIR=/var/run/docker/netns
+BRIDGEIF=br0
+
+function die {
+    echo $*
+    exit 1
+}
+
+type -P ${DOCKER} > /dev/null || die "This tool requires the docker binary"
+type -P ${NSENTER} > /dev/null || die "This tool requires nsenter"
+type -P ${BRIDGE} > /dev/null || die "This tool requires bridge"
+type -P ${BRCTL} > /dev/null || die "This tool requires brctl"
+type -P ${IPTABLES} > /dev/null || die "This tool requires iptables"
+
+echo "iptables configuration"
+${IPTABLES} -n -v -L -t filter
+${IPTABLES} -n -v -L -t nat
+echo ""
+
+echo "Overlay network configuration"
+for networkID in $(${DOCKER} network ls --filter driver=overlay -q) ; do
```

Comment:
I create an overlay network with id `0qv3qep6jxnn`. There is no under `/var/run/docker/netns` matches it. Do I miss something?

```
root@ip-172-19-241-147:~# docker network ls --filter driver=overlay -q | cut -c1-10
amj1vcvggz
0qv3qep6jx
root@ip-172-19-241-147:~# ls /var/run/docker/netns/
1-amj1vcvggz  8d598d017f73  b07c0f8fd4b9  eb32cb61cce8
```